lldb/tools/lldb-vscode/package.json

```diff
@@ -240,6 +240,16 @@
 							"timeout": {
 								"type": "string",
 								"description": "The time in seconds to wait for a program to stop at entry point when launching with \"launchCommands\". Defaults to 30 seconds."
+							},
+							"descriptiveSummaries": {
+								"type": "boolean",
+								"description": "Show more descriptive variable summaries, which might incur in worse performance.",
```

Comment:
I don't think "incur in" is correct, I think you incur something, like a performance penalty. Maybe simply saying "at the cost of performance" or "could/might decrease performance"?

Reply:
Thanks for the feedback.